linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml

```diff
@@ -24,7 +24,7 @@ references:
     srg@rhel6: SRG-OS-000121,SRG-OS-000123
     disa: 16,1682
     nist: AC-2(2),AC-2(3)
-    srg: "2"
+    srg: SRG-OS-000002
```

Comment:
@cipherboy this SRG isn't right as it's the legacy SRG.

Reply:
So do I change this to `SRG-OS-000002-GPOS-00153`?